### PR TITLE
[main] UI text rview changes-EOS-14222

### PR DESCRIPTION
**Pull request by namrata-khake-ui**
# UI

text review changes

## Problem Statement
<pre>
  <code>
    Story Ref (if any):EOS-14222
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  No
  </code>
</pre>
## Problem Description
<pre>
  <code>
1. On every account creation and access key popup warning text should changed from
"Save this information, you will not see it again. Download as CSV and close" to
"Click Download and close to save this information in CSV format."
2. All the column which are having any action edit or delete for those header name should be "Action"
3. Remove extra colon from text "I agree to the following:: "
Firmware Update 
4. Remove <br> firmware update text

**The below changes Pull request merged for Cortx1.0-dev branch. Added changes for main branc collectlively** 
https://github.com/Seagate/cortx-management-portal/pull/165

1. Welcome!!! changed to Welcome!
2. Changed "Click on Start to proceed with on onboarding steps." to "Click Start to proceed with onboarding steps."
3. username info tooltip : one space before this sentence "The username must not be 'root' or 'Root'."
4.password info tooltip: change from "Password must contain: Minimum 8 characters, One uppercase letter, One lowercase letter, One special character, One number"
   to "Password must contain: Minimum 8 characters, One uppercase letter, One lowercase letter, One special character, and One number"
5.DNS setting
    there should be little bit space above "DNS server" and "search domains" inputs
  

  </code>
</pre>
## Solution/Screenshots
image
image
image
image
image
image
image
image
image
image
image

<pre>
  <code>
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
  tested on locally
  </code>
Signed-off-by: Namrata Khake <namrata.khake@seagate.com>

### CODE REVIEW

File: gui/src/common/regex-helpers.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const passwordRegex = helpers.regex(
 );
 export const passwordTooltipMessage =
   // tslint:disable-next-line
-  "Password must contain: Minimum 8 characters, One uppercase letter, One lowercase letter, One special character, One number";
+  "Password must contain: Minimum 8 characters, One uppercase letter, One lowercase letter, One special character, and One number";
```
Comment (shrimetta) [inline gui/src/common/regex-helpers.ts:44]: Please remove comma before and

**Review by shrimetta — CHANGES_REQUESTED** [review]
Minor typo fix

File: gui/src/locales/en.json [hunk 2]
```diff
@@ -258,7 +259,8 @@
     "searchDomainRequired": "Search domain is required.",
     "DNSResolverSettings": "DNS resolver settings",
     "updateFirmware": "Update firmware",
-    "updateStorageEnclosure": "Update the storage enclosure firmware. Upload the firmware bundle file (.bin) received from Seagate. <br> Click Start update once the bundle file is uploaded successfully.",
+    "updateStorageEnclosure": "Update the storage enclosure firmware. Upload the firmware bundle file (.bin) received from Seagate.",
+    "updateStorageEnclosureClick": " Click Start update once the bundle file is uploaded successfully.",
```
Comment (shrimetta) [inline gui/src/locales/en.json:263]: Please remove space before Click

File: gui/src/common/regex-helpers.ts (hunk 1 above)
Comment (namrata-khake-ui) [reply gui/src/common/regex-helpers.ts:44]: that was the requirement, document team suggested to add comma

File: gui/src/locales/en.json (hunk 2 above)
Comment (namrata-khake-ui) [reply gui/src/locales/en.json]: fixed this review comment

**Review by JayshreeAmore — APPROVED** [review]

**Review by giftedUI — APPROVED** [review]

**Review by shrimetta — APPROVED** [review]

File: gui/src/components/s3/bucket-creation.vue [hunk 3]
```diff
@@ -384,6 +383,10 @@ export default class CortxBucketCreation extends Vue {
         text: "Name",
         value: "name",
         sortable: false
+      },
+      { 
+        text: "Action",
```
Comment (giftedUI) [inline gui/src/components/s3/bucket-creation.vue:388]: Can we use `i18n.t("common.action")` here instead of plain text?

Same in other places.

**Review by giftedUI — CHANGES_REQUESTED** [review]

File: gui/src/components/s3/bucket-creation.vue (hunk 3 above)
Comment (namrata-khake-ui) [reply gui/src/components/s3/bucket-creation.vue]: resolved above review comment

**Review by giftedUI — APPROVED** [review]

**Review by kupranay — APPROVED** [review]

**Pull request merged**